dev-db/sadisplay/sadisplay-0.4.9.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="SqlAlchemy schema display script"
+HOMEPAGE="https://bitbucket.org/estin/sadisplay/wiki/Home"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-python/sqlalchemy[${PYTHON_USEDEP}]"
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	"
+
+distutils_enable_tests nose
```

Comment:
`Failure: ModuleNotFoundError (No module named 'pytest') ... ERROR`

You can still add test dependencies while using `distutils_enable_tests nose` if it really requires both. I tested. after installing pytest everything works.